doc/Admin Documentation.md

```diff
@@ -35,7 +35,10 @@ occ twofactorauth:gateway:configure signal
 Url: https://www.telegram.org/
 Stability: Unstable
 
-In order to send messages via the Telegram network, you have to register a [Telegram Bot](https://core.telegram.org/bots), which is used to send authentication codes to users after they have initiated a conversation and entered their Telegram ID.
+This gateways allows you to send messages via the Signal protocol. In order to send messages,
```

Comment:
Signal != Telegram

On a general note: please don't use a single PR for unrelated changes. It's better to have this separate, e.g. one of the CLI argument description and another one for the documentation update. Thanks :raised_hands:

Reply:
Oh, sorry I mixed-up some buttons. I wasn't aware that I've directly commited something. I was still in the process to create a new pull request. Could you cancel it so that I can do it the right way?

Reply:
No worries :)

What do you mean with cancel? You can also "fix" this PR by changing the commits and force pushing those changes. Do you work with git locally or the github web interface?

Reply:
I'm using the github web interface. I'm not so familiar with all the functions and so I the changes to the documentation were added to my initial pull request.
Before I generate more confusion please remove the document changes and I will try to generate a new pull request for it (although this was my initial intension) Or would it be sufficient to delete the following commits:
![grafik](https://user-images.githubusercontent.com/2572764/50779358-63351000-12a0-11e9-9704-73076c6e64ee.png)

Reply:
Okay. So what you wanted to do is create commits on two different branches. But they ended up on the same branch.

I think the problem is that you are commiting to `master` directly.

If you want I can either try to show you how to fix this (not sure if possible via the web gui) or just fix it for you and split the PR on my side :)

Reply:
It is never to late to teach an old dog new tricks ;-)

Reply:
You're right ;)

This is the current history:

```
$ git log --oneline 
01afd1c (HEAD, j-ed/master) Improved formatting.
409663d fixed typo.
0a0a4be Created an initial user documentation.
aabd194 Improved description of Telegram authentication gateway setup process.
63d6519 Improved description of Telegram authentication gateway setup process.
763a210 improved the gateway and identifier argument description.
4530e35 improved gateway argument description.
3a0c999 (origin/master, origin/HEAD, master) [tx-robot] updated from transifex
...
```

Hence, 763a210 is the last commit of the first fix. We can create a branch off that:

```
$ git checkout 763a210 -b fix/cli-argument-description
```

The history should be clean:
```
$ git log --oneline 
763a210 (HEAD -> fix/cli-argument-description) improved the gateway and identifier argument description.
4530e35 improved gateway argument description.
3a0c999 (origin/master, origin/HEAD, master) [tx-robot] updated from transifex
...
```

For the second part, we have to cheat a bit. We can't have the latest commits without the first ones without some more tricks because they are dependent. But you can create a new branch for it:

```
$ git checkout master -b fix/admin-docs-typos
```

You can push these branches with

```
$ git push origin fix/cli-argument-description
$ git push origin fix/admin-docs-typos
```

... and create PRs for them on the web interface.

Reply:
Ok, these are the commands on the git command line. So if I want to use the webgui I could also e.g. delete my previous fork of the repository, create a new one, apply my changes to the documentation to it and create a new pull request, right?

Reply:
Ah yes, that should be possible. When you save/commit the changes make sure to select a new branch :wink:

Reply:
So, next try. I hope it looks better now https://github.com/nextcloud/twofactor_gateway/pull/179 ?!

Reply:
It does! Thanks!

You also have to re-submit the argument description fixes because this pull request now conflicts with the newly submitted documentation updates.

Reply:
Ok, done with https://github.com/nextcloud/twofactor_gateway/pull/180.

Reply:
Thanks a lot :)